objects/Handlebars.js

```diff
@@ -22,26 +22,25 @@ export default class Handlebars extends CustomObject {
         this.simulateStop = false;
     }
 
-    render(context, program_state, t, a, simulateStop, model_transform=Mat4.identity()) {
+    render(context, program_state, t, a, simulateStop, num_handlebars = 15, spacing = 3.5, bar_length = 20, model_transform=Mat4.identity()) {
         let i = 0;
 
         const stick_width = 0.1; 
         const stick_length = 1;
         const loop_width = 0.4;
         const loop_thick = 0.15;
-        const barLength = 20;
-        const barWidth = 0.15;
+        const bar_width = 0.15;
         
```

Comment:
To evenly space out the handles, we'd use the formula below:

```suggestion
        const spacing = bar_length / (num_handlebars + 1);
```